src/mtev_cluster.c

```diff
@@ -65,13 +82,16 @@ struct mtev_cluster_t {
   mtev_cht_t **cht;
   mtev_net_heartbeat_ctx *hbctx;
   mtev_cluster_node_t *oldest_node;
+  struct jl_array_list *hb_payloads;
```

Comment:
If we toss json-lib for an alternative implementation, those structs would go away.. I'd rather not see reliance on those structures.  I see no reason for the arrays to every shrink... so this seems like a trivial use of plain C arrays...  find first null to insert, grow to accommodate, null to remove.  Also, I do not think the  mtev_hash will be poor performance.